### PR TITLE
DHFPROD-7878: Saving coordinates of all entity nodes in just one api call

### PR DESCRIPTION
**Pull request by xnikhil08**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [N/A] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [N/A] Reviewed Tests

### CODE REVIEW

**Review by wooldridge — DISMISSED** [review]
I'm seeing the single call being made on node statilbization, nice!

single-call-stabilization

**Review by devinnicholson — APPROVED** [review]
The loop is fixed and I'm seeing one call now. Looks good

**Review by AkshayBajajML — DISMISSED** [review]

File: marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-vis/graph-vis.tsx [hunk 1]
```diff
@@ -124,21 +117,49 @@ const GraphVis: React.FC<Props> = (props) => {
...
+  const saveUnsavedCoords = async () => {
     if (props.entityTypes) {
       let newCoords = {...coords};
+      let updatedEntityDefinitions: any = [];
       props.entityTypes.forEach(ent => {
         if (!coordsExist(ent.entityName)) {
           let positions = network.getPositions([ent.entityName])[ent.entityName];
           newCoords[ent.entityName] = {graphX: positions.x, graphY: positions.y};
-          props.saveEntityCoords(ent.entityName, positions.x, positions.y);
+          let entityDefinition: EntityModified = getDefinitionsPayload(ent.entityName, {graphX: positions.x, graphY: positions.y}, ent);
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-vis/graph-vis.tsx:128]: Suggestion: if you need to have ent and ent.entityName, you can just have ent and use ent.entityName inside the function.

Comment (xnikhil08) [reply marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-vis/graph-vis.tsx]: Done.

**Review by aniamy — APPROVED** [review]

**Review by AkshayBajajML — APPROVED** [review]

**Pull request merged**